docs/source/concepts/bento.rst

```diff
@@ -728,6 +728,26 @@ Python Options Table
 | wheels            | List of paths to wheels to include in the bento                                    |
 +-------------------+------------------------------------------------------------------------------------+
 
+Models
+^^^^^^
+
+You can specify the model to be used for building a bento using a string model tag or a dictionary, which will be written to the ``bento.yaml`` file in the bento package.
+These configurations are used to fetch models from Yatai. You can download models to your local model store by running ``bentoml models pull``.
```

Comment:
```suggestion
When starting from an existing project, these configurations are also used to fetch models from Yatai. You can download models to your local model store by running ``bentoml models pull``.
```

Just a little clarification.

Reply:
"When starting from an existing project" means users can download models using these configs and create their own projects on top of these downloaded models. Is it correct?

Reply:
It means someone clones the project from, such as, GitHub and hasn't got the models yet, because the models are usually trained by another group of people and uploaded to the cloud.

Reply:
Updated.